retry_test.go

```diff
@@ -92,3 +94,16 @@ func TestFixedSleep(t *testing.T) {
 	assert.Error(t, err)
 	assert.True(t, dur < 500*time.Millisecond, "3 times default retry is shorter then 500ms")
 }
+
+func TestLastErrorOnly(t *testing.T) {
+	var retrySum uint
+	err := Do(
+		func() error { return errors.New(fmt.Sprintf("%d", retrySum)) },
+		OnRetry(func(n uint, err error) { retrySum += 1 }),
+		Delay(time.Nanosecond),
+		LastErrorOnly(true),
+	)
+	if err.Error() != "9" {
```

Comment:
please use assert library
```
assert.Equal(t, "9", err.Error())
```
this looks more readable for me - thanks

Reply:
done